### PR TITLE
Sorting Storybook stories

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] DX

## Description

This introduces properly sorted Storybook stories. Here is the [Storybook for this PR](https://deploy-preview-8058--storybookdevto.netlify.app/?path=/story/1-guidelines--crayons)

The PR looks intimidating at 45 files, but it's simply renaming Storybook story names for the tree view in Storybook.

e.g.

```
export default {
  title: 'App Components/Comment Subscription',
  decorators: [withKnobs],
};
```

becomes
```
export default {
  title: '4_App Components/Comment Subscription',
  decorators: [withKnobs],
};
```

so that App Components appear 4th at the top level of the Storybook treeview.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

image

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

The Goodlife characters referencing the sorting hat from Harry Potter

### CODE REVIEW

**Comment by nickytonline** [comment]
It looks intimidating at a 45 file PR, but it's simply renaming Storybook story names for the tree view in Storybook.

File: app/javascript/.storybook/preview.js [hunk 1]
```diff
@@ -50,3 +50,12 @@ const themeSwitcherDecorator = (storyFn) => (
 );
 
 addDecorator(themeSwitcherDecorator);
+
+addParameters({
```
Comment (nickytonline) [inline app/javascript/.storybook/preview.js:54]: This is what does the sorting. See the documentation for [Sorting Stories](https://storybook.js.org/docs/configurations/options-parameter/#sorting-stories) for more info.

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-12.16
\ No newline at end of file
+12.16.0
```
Comment (rhymes) [inline .nvmrc:1]: I have 12.16.3 locally, isn't this going to mess with minor versions?

Comment (nickytonline) [reply .nvmrc:1]: I will remove this. I was trying to fix the broken build.

**Comment by nickytonline** [comment]
@vaidehijoshi, any reason Percy is failing when there have been no UI changes? In some cases it says snapshots have been changed and for others, they just need to be approved. Either way I do not have access to approve even though I accepted the Percy invitation.

image

**Comment by vaidehijoshi** [comment]
@nickytonline the diffs in that build are intermittent (something I'm trying to fix as I find the time).

You _should_ be able to approve, though!! Can you ping me and show me what you're seeing when you log into Percy?

**Comment by nickytonline** [comment]
> @nickytonline the diffs in that build are intermittent (something I'm trying to fix as I find the time).
> 
> You _should_ be able to approve, though!! Can you ping me and show me what you're seeing when you log into Percy?

It's all good. I can approve now. Just a problem in front of the keyboard. 🤪

**Pull request merged**